src/conda/install.sh

```diff
@@ -69,12 +69,12 @@ if ! conda --version &> /dev/null ; then
     usermod -a -G conda "${USERNAME}"
 
     # Install dependencies
-    check_packages curl ca-certificates gnupg2
+    check_packages curl ca-certificates
 
     echo "Installing Conda..."
 
-    curl -sS https://repo.anaconda.com/pkgs/misc/gpgkeys/anaconda.asc | gpg --dearmor > /usr/share/keyrings/conda-archive-keyring.gpg
```

Comment:
I think storing keyring as `.gpg` is intentional [details](https://wiki.debian.org/DebianRepository/UseThirdParty#OpenPGP_key__handling).  

<img width="1725" alt="image" src="https://user-images.githubusercontent.com/24955913/200446815-a280c91e-0344-4427-b9f5-01293df70d8f.png">

`.gpg` uses a binary format and `.asc` an ASCII-armored signature with or without wrapped document, plain text format

@Chuxel could you help confirm if that's the case?